examples/oft/hardhat.config.ts

```diff
@@ -75,4 +99,57 @@ const config: HardhatUserConfig = {
     },
 }
 
+// TODO this will only work if the safe url is always one of the supported networks here https://docs.safe.global/core-api/transaction-service-supported-networks
+// which seems to be the case for ZROClaim at least
+const validateSafeConfig = async (config: any): Promise<boolean> => {
+    assert(config.safeAddress != null, 'Missing safeAddress')
+    assert(config.safeUrl != null, 'Missing safeUrl')
+
+    // Construct the API URL to query the Safe's balance
+    const apiUrl = `${config.safeUrl}/api/v1/safes/${config.safeAddress}/balances/`
+    // const apiUrl = `${config.safeUrl}/api/v1/safes/${config.safeAddress}/`; // TODO or should we use this general info api endpoint
```

Comment:
See todo -- which api url is better for validating? I am assuming that it would be the general info api endpoint instead of this one that checks the balances. Also perhaps we should be using their `api/v2`?